public/app/core/services/context_srv.ts

```diff
@@ -19,7 +19,6 @@ export class User implements Omit<CurrentUserInternal, 'lightTheme'> {
   name: string;
   externalUserId: string;
   theme: string;
-  orgCount: number;
```

Comment:
This is a public interface used by legacy Angular plugins. Are we sure we want to remove this property? It will be a breaking change for any plugin that uses this. (If there are any plugins out there that do use it.)

Reply:
From the auth side we want to slim down SignedInUser and reduce the amount of information exposed directly through request context.

We can keep this frontend interface working with an extra DB call from index.go if you think it's valuable for plugin developers, wdyt?

Reply:
Reverted to old frontend behavior, it still accomplishes the main objective of the PR. If you ever feel it can be removed, happy to help